src/components/NavBar/NavBar.scss

```diff
@@ -44,3 +47,92 @@ li {
 .color-yellow{
 	color:var(--yellow);
 }
+.mobview{
+  margin: auto;
+  width: 100%;
+  justify-content: center;
+}
+.mobview .container{
+  display: flex;
+  flex-direction: column;
+}
+.mobview .container .header{
+  font-size: 20px;
+	font-family: inherit;
+	font-weight: 500;
+  display: flex;

```

Comment:
we might use flex regularly, so can you create a Flex component which can be used as common component?